src/options/get-options-from-file.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import path from 'path';
+
+import { Options } from './index';
+
+const configFileName = '.filenamelintrc';
+
+function isFileEmpty(content: string): boolean {
+  return content.trim() === '';
+}
+
+export default function getOptionsFromFile(): Partial<Options> {
+  const configFilePath = path.join(process.cwd(), configFileName);
+
+  if (fs.existsSync(configFilePath)) {
+    const content = fs.readFileSync(configFilePath, 'utf8');
+
+    if (isFileEmpty(content)) {
+      return {};
+    }
+
+    const options = JSON.parse(content);
```

Comment:
Maybe `try ... catch` and handle this error gracefully? I'd be better than `unexpected token at line...`

Reply:
nice catch, i will do it